mx.py

```diff
@@ -1713,15 +1716,21 @@ def _getCompiler(self):
         if self.args.jdt and not self.args.force_javac:
             return ECJCompiler(self.args.jdt, self.args.extra_javac_args)
         else:
-            return JavacCompiler(self.args.alt_javac, self.args.extra_javac_args)
+            if self.args.no_javac_daemon:
+                return JavacCompiler(self.args.alt_javac, self.args.extra_javac_args)
+            else:
+                return JavacCompilerDaemon(self.args.alt_javac, self.args.extra_javac_args)
```

Comment:
It's about using a different javac than the one from the JDK.
I don't think it's used anywhere. Maybe we should just remove the support.